archivist/.gitignore

```diff
@@ -0,0 +1,2 @@
+# General
+.DS_Store
```

Comment:
We already have this for the whole repo, I'm not sure about shipping one specific for the theme

Reply:
Ah ok happy to remove it then.